.travis.yml

```diff
@@ -4,7 +4,6 @@ go:
     - tip
     - 1.5.1
     - 1.5
-    - 1.4.3
```

Comment:
Should we support every patch level or just the latest in every minor? I know go is pretty stable and this is really low effort but still. It might not be necessary. 
 
I think at some point 1.4.3 should be supported as well. :smile: